app-editors/komodo-edit-bin/komodo-edit-bin-10.2.2-r1.ebuild

```diff
@@ -44,29 +43,33 @@ RDEPEND="${DEPEND}
 	gnome-base/libgnomeui
 	gnome-base/libgnome"
 
-RESTRICT="mirror"
-
 QA_PREBUILT="*"
 QA_FLAGS_IGNORED="*"
 QA_PRESTRIPPED="*"
 
-S="${WORKDIR}/Komodo-Edit-${PV}-${MY_BUILD}-linux-${ARCH/amd64/x86_64}"
+src_unpack() {
+	if use amd64; then
+		S="${WORKDIR}"/Komodo-Edit-"${PV}"-"${MY_BUILD}"-linux-x86_64
+	fi
+
+	if use x86; then
+		S="${WORKDIR}"/Komodo-Edit-"${PV}"-"${MY_BUILD}"-linux-x86
+	fi
+}
 
 src_install() {
 	if use pax_kernel; then
 		pax-mark m "${S}"/INSTALLDIR/lib/mozilla/komodo
 	fi
 
-	insinto /opt/${PN}
+	insinto /opt/"${PN}"
 	doins -r "${S}"/INSTALLDIR/share
 	mv "${S}"/INSTALLDIR/lib "${ED}opt/${PN}/" || die "Installation failed"
 
-	make_wrapper komodo ${EPREFIX}/opt/${PN}/lib/mozilla/komodo ${EPREFIX}/opt/${PN}/lib/mozilla /usr/bin
-
-	doicon "${S}/INSTALLDIR/share/icons/komodo48.png"
```

Comment:
This was due to package collision warnings from Komodo-IDE, the path is set in the desktop entry. I guess I could rename the icon, then doicon.